vendor/bat-native-ads/src/bat/ads/internal/base/unittest/unittest_command_line_switch_util.cc

```diff
@@ -0,0 +1,67 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/base/unittest/unittest_command_line_switch_util.h"
+
+#include "base/check.h"
+#include "base/command_line.h"
+#include "base/strings/strcat.h"
+#include "bat/ads/internal/base/strings/string_strip_util.h"
+#include "bat/ads/internal/base/unittest/unittest_string_util.h"
+
+namespace ads {
+
+namespace {
+
+constexpr char kNoCommandLineSwitchKey[] = "WithNoCommandLineSwitch";
+constexpr char kNoCommandLineSwitchValue[] = "EmptyValue";
+
+std::string CommandLineSwitchToString(
+    const CommandLineSwitchInfo& command_line_switch) {
+  const std::string switch_value = command_line_switch.value.empty()
+                                       ? kNoCommandLineSwitchValue
+                                       : command_line_switch.value;
+
+  return base::StrCat({command_line_switch.key, "=", switch_value});
+}
+
+std::string SanitizeCommandLineSwitchFromString(
+    const std::string& command_line_switch) {
+  return CapitalizeFirstCharacterOfEachWordAndTrimWhitespace(
+      StripNonAlphaNumericCharacters(command_line_switch));
+}
+
+}  // namespace
+
+void AppendCommandLineSwitches(
+    const CommandLineSwitchList& command_line_switches) {
+  if (command_line_switches.empty()) {
+    return;
+  }
+
+  CHECK(base::CommandLine::InitializedForCurrentProcess);
```

Comment:
Braces are missing. Should be:
CHECK(base::CommandLine::InitializedForCurrentProcess());

Reply:
Fixed